surface/src/3rdparty/opennurbs/opennurbs_3dm_settings.cpp

```diff
@@ -756,6 +754,8 @@ void ON_3dmAnnotationSettings::Default()
   m_world_view_hatch_scale = 1.0f;
   m_bEnableAnnotationScaling = 1;
   m_bEnableHatchScaling = 1;
+
+  memset(m_reserved, 0, sizeof(m_reserved));
 }
```

Comment:
I compare all variables in this `Default` function, and make sure all member variables are set to default values. Except existed assignments, I add `memset` for `m_reserved` referred to these in other `Default` functions:
https://github.com/PointCloudLibrary/pcl/blob/6555d8a6f10f0d2fa6f60e99e10360141ad57a89/surface/src/3rdparty/opennurbs/opennurbs_3dm_settings.cpp#L444
https://github.com/PointCloudLibrary/pcl/blob/6555d8a6f10f0d2fa6f60e99e10360141ad57a89/surface/src/3rdparty/opennurbs/opennurbs_3dm_settings.cpp#L485